### DISCUSSION TITLE
Paste in text only

### DISCUSSION THREAD

**Discussion by XternalSoft** [awcodes/filament-tiptap-editor#171] [Ideas]
Hello,

It would be really great, in version 3, to be able to configure paste to be text only.

Do you think this could be possible?

**Comment by awcodes** [comment]
Not entirely sure how that would work. It would have to be all or nothing. I can't think of a way to conditionally choose to paste something with links or not, for example.

**Reply by XternalSoft** [reply]
Yes, it has to be all or nothing. 
I had made the change in your v2. I had added "paste_as_text:true" in the configuration. 

Because I have users who copy and paste from different sources and this adds bad styles.

**Reply by awcodes** [reply]
where were you defining the config?

**Reply by XternalSoft** [reply]
Oops... 

I've just realized that I've mixed up 2 plugins... with Filament v2 I use https://github.com/mohamedsabil83/filament-forms-tinyeditor and I must have forgotten (it's no good doing several things at once) that I'd tried your plugin.

So, with tinyeditor there's an option to enable paste as text.  BUT it would be really nice to have the same kind of functionality in your plugin :)

So, I'll leave you to appreciate (or not) the idea ;)

**Reply by zakmcrbit** [reply]
+1 Is there a solution for this? I urgently need this too.
